util/create.rsid.sqlite3.py

```diff
@@ -0,0 +1,61 @@
+import glob
+import gzip
+import os.path
+import sqlite3
+import subprocess
+
+# Generate shell script to make bgzipped files with chrom, pos, ref, alt, and rsnum
+
+tsvList = glob.glob("../data/ebi_ge/*/*.tsv.gz")
+temp = subprocess.call(["mkdir", "-p", "temp"])
+outfileList = []
+
+with open("./temp/get.rsid.tables.sh", "w") as w:
+    for infile in tsvList:
+        outfile = os.path.join(
+            "./temp",
+            os.path.basename(infile).replace(".tsv.gz", ".rsid.tsv.gz"),
+        )
+        outfileList.append(outfile)
+        w.write(
+            f"zcat {infile} | cut -f 4-7,21 | sort | uniq | bgzip -c > {outfile}\n"
+        )
+
+
+# Run script
+
+temp = subprocess.call(["bash", "./temp/get.rsid.tables.sh"])
+
+
+# create SQLite3 database of rsids
+
+conn = sqlite3.connect("../data/rsid.sqlite3.db")
+
+
+def parseline(line):
+    (chrom, pos, ref, alt, rsid) = line.decode("utf-8").rstrip("\n").split()

```

Comment:
Random helpful note: you can use `gzip.open(filename, 'rt')` to open a file as text mode, which gets you out of needing to explicitly `.decode('utf-8')` all your strings.